### PR TITLE
fixed protein-translation.spec.coffee

### PR DESCRIPTION
**Pull request by ladokp**
rewrote the last test to check for an exception and not a string like seen in other exercise tests.

Related to #259

### CODE REVIEW

**Comment by ladokp** [comment]
@ErikSchierboom: Opened a new PR with my current try to solve the broken test for protein-translation.

File: exercises/practice/protein-translation/protein-translation.spec.coffee [hunk 1]
```diff
@@ -134,4 +134,4 @@ describe 'ProteinTranslation', ->
   xit "Non-existing codon can't translate", ->
     expect ->
       ProteinTranslation.proteins("AAA")
-    .toThrow "Invalid codon"
\ No newline at end of file
+    .toThrow new Error("Invalid codon")
```
Comment (ErikSchierboom) [inline exercises/practice/protein-translation/protein-translation.spec.coffee:137]: As suggested in https://github.com/exercism/coffeescript/pull/265#issuecomment-1919713374, maybe use:

```suggestion
    .toThrowError('Invalid codon')
```

Comment (ladokp) [reply exercises/practice/protein-translation/protein-translation.spec.coffee]: The checks failed with this change, looks like this functions has two parameters.

**Comment by ErikSchierboom** [comment]
Are we using an outdated version of the matchers library or something?

**Comment by ladokp** [comment]
Maybe, let's stick to the "toThrow()" function for now?

**Comment by ErikSchierboom** [comment]
I guess we have to

**Comment by ErikSchierboom** [comment]
Thanks!

**Comment by ladokp** [comment]
Thank you too, I hope the exercise is solvable now.

**Pull request merged**